lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -307,12 +315,23 @@ def stub_email_alert_api_does_not_have_subscriber_list_by_slug(slug:)
       alias_method :email_alert_api_creates_a_subscription, :stub_email_alert_api_creates_a_subscription
       alias_method :email_alert_api_creates_an_existing_subscription, :stub_email_alert_api_creates_an_existing_subscription
       alias_method :email_alert_api_refuses_to_create_subscription, :stub_email_alert_api_refuses_to_create_subscription
-      alias_method :email_alert_api_creates_an_auth_token, :stub_email_alert_api_creates_an_auth_token
       alias_method :email_alert_api_has_subscriber_list_by_slug, :stub_email_alert_api_has_subscriber_list_by_slug
       alias_method :email_alert_api_does_not_have_subscriber_list_by_slug, :stub_email_alert_api_does_not_have_subscriber_list_by_slug
+      alias_method :stub_email_alert_api_creates_an_auth_token, :stub_email_alert_api_sends_subscriber_verification_email
+      alias_method :email_alert_api_sends_subscriber_verification_email, :stub_email_alert_api_sends_subscriber_verification_email
 
     private
 
+      def get_subscription_verification_response(id, address, topic_id)
```

Comment:
I don't think this is right. Comparing [with the controller action](https://github.com/alphagov/email-alert-api/blob/master/app/controllers/subscriptions_auth_token_controller.rb#L13), I think we just return an empty hash.

Reply:
Also, the status code is 200/OK.